### PR TITLE
[example] [refactor] Rename some old examples to avoid confusing people

### PR DESCRIPTION
**Pull request by archibate**
Related issue = #... (if any)

[[Click here for the format server]](http://kun.csail.mit.edu:31415/)

### CODE REVIEW

**Comment by rexwangcc** [comment]
lgtm and thanks for these enhancements! 

My 2 cents:
- Personally as i was ramping up, what confused me a bit was that 'mnist.py', instead of a mnist classifier, was simply a "mnist dataset downloader". Even it's obvious to figure out by looking at the code, it'd still be great for users to make it clear either thru names or in docstrings.
- Personally, ml_mlp(multilayer perceptron) is a bit clearer than ml_test, but i could be subjective here.

**Comment by archibate** [comment]
> lgtm and thanks for these enhancements!
> 
> My trivial 2 cents:
> 
>     * Personally as i was ramping up, what confused me a bit was that 'mnist.py', instead of a mnist classifier, was simply a "mnist dataset downloader". Even it's obvious to figure out by looking at the code, it'd still be great for users to make it clear either thru names or in docstrings.
> 
>     * Personally, ml_mlp(multilayer perceptron) is a bit clearer than ml_test, but i could be subjective here.

OIC, sorry for my ignorance on ML.. let's use `ml_multilayer_perceptron.py`, wdyt?

File: examples/minimal.py [hunk 1]
```diff
@@ -1,9 +0,0 @@
-import taichi as ti
```
Comment (yuanming-hu) [inline examples/minimal.py:1]: This is actually a useful file. Let's move it to `misc`.

Comment (archibate) [reply examples/minimal.py:1]: But we already have minimal_timed.py?

**Pull request merged**